sktime/classification/__init__.py

```diff
@@ -0,0 +1,2 @@
+# -*- coding: utf-8 -*-
+"""Classifier Tests."""
```

Comment:
This is for the whole classification package, right? Not just tests.

A bit of a nitpick.

Reply:
Good spot, tganks